DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Package: workflowscriptscommon
 Type: Package
 Title: Common Functions for Workflow Script Packages
-Version: 0.0.2
+Version: 0.0.3
```

Comment:
We seem to have not updated the version file in previous releases (see the releases), but this should be 0.0.5 now